### DISCUSSION TITLE
Keeps Closing on admin run and normal run.

### DISCUSSION THREAD

**Discussion by N0AH-D3V** [SteamRE/DepotDownloader#440] [Q&A] [answered]
### What did you expect to happen?

i thought it would open and i could install the older version of the game i was trying to downgrade.

### Instead of that, what actually happened?

i thought it would open and i could install the older version of the game i was trying to downgrade. it just immediately closed. it opened one time but i tried to paste and it closed again

### Which operating system are you running on?

Windows

### Version

Latest

**Comment by psychonic** [comment, marked as answer]
Depot Downloader is a console application. There is no gui. If you do not pass any command line parameters, it prints usage info and exits.

**Comment by hemthet** [comment]
I had this same problem. What you have to do is go to windows settings and look up terminal settings. Scroll all the way down till you see the word "Terminal". Switch it from Windows Console Host to Windows Terminal. This should fix the problem of it immediately closing.

**Reply by N0AH-D3V** [reply]
Thanks

**Reply by ABakedWaffle** [reply]
I have the same problem, so I just tried that and it didn't work. It still closed after opening

**Comment by u2020bullet** [comment]
Same issue as other people. Even with the terminal settings applied, it still only opes for only a millisecond.

**Comment by Luckz** [comment]
Maybe a picture-heavy guide like https://steamcommunity.com/sharedfiles/filedetails/?id=2353930763 could help.
